webui/react/src/hooks/useCreateModelModal.tsx

```diff
@@ -80,7 +80,6 @@ const useCreateModelModal = (): ModalHooks => {
         labels: tags,
         metadata: metadata,
         name: modelName,
-        username: user?.username,
```

Comment:
**question:** just checking to see if is ok to create a model without a user id. Does it figure it out on the backend based on the auth token of who is creating a model?

Reply:
yep, it checks `CurrentUser`